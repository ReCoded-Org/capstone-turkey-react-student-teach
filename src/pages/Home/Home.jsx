import HomeContent from '../../components/home/HomeContent';

function Home() {
  return <HomeContent />;
}

export default Home;
