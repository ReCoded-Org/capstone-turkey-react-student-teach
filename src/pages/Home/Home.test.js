import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from './Home';

it('Navbar snapshot', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Home />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
