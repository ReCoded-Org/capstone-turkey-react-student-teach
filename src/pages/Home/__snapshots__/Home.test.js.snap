// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot 1`] = `
<div>
  Home
</div>
`;
